modules/battery_manager/ajax/add_or_edit_entry.php

```diff
@@ -0,0 +1,207 @@
+<?php
+/**
+ * Battery Manager entry adder and editer
+ *
+ * Checks Test Battery for duplicates
+ * Handles insertions and updates in the Test Battery
+ *
+ * PHP Version 7
+ *
+ * @category Loris
+ * @package  Battery_Manager
+ * @author   Victoria Foing <victoria.foing@mcin.ca>
+ * @license  Loris license
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+
+// Determine which action is called
+if (isset($_GET['action'])) {
+    $action = $_GET['action'];
+    if ($action == "checkForDuplicate") {
+        echo checkForDuplicate();
+    } else if ($action == "add") {
+        addEntry();
+    } else if ($action == "edit") {
+        editEntry();
```

Comment:
Add and edit shouldn't be done via a get request, they should be done via a post or put

Reply:
I concur